examples/gross_pitaevskii_custom_potential.jl

```diff
@@ -0,0 +1,114 @@
+# # Gross-Pitaevskii equation in one dimension with forces and custom potential
+
+# We solve the 1D Gross-Pitaevskii equation with a custom potential.
+# This is similar to the
+# previous example ([Gross-Pitaevskii equation in one dimension](@ref)) where
+# computing forces was not possible as we used an `ExternalFromReal` term
+# instead of an `AtomicLocal` term (see [`Model` datastructure](@ref)).
+# Here, we use `ElementCustomIonPotential`
+# with a custom potential to generate an `AtomicLocal` term for which we can
+# compute forces.
```

Comment:
Instead of turning that negative I would simply say that in this example you're going to show how to define local potentials attached to atoms, which allows reusing the machinery for atoms (forces, etc)